### ISSUE TITLE
Same Aria label is defined for two different list box 'Get' and 'v1.0' on Graph Explorer page.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#751]
**Describe the bug**
User who rely on screen reader will face difficulty if same aria-label is provided for for two different list box.

**To Reproduce**
Step 1: Open 	Graph Explorer - Microsoft Graph URL in Anaheim dev  Edge browser.
Step 2: Navigate to the list box 'Get' and 'v1.0'.
Step 3: Observe the issue.

Same Aria label is defined for two different  list box 'Get' and 'v1.0' on Graph Explorer page.

**Expected behavior**
Aria label should be unique for different controls.

**Desktop (please complete the following information):**
OS: OS build Version 2004 (OS Build 20170.1000)
URL: Graph Explorer - Microsoft Graph
Browser: Anaheim Edge Dev Version: 87.0.634.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Additional context**
